fastapi-google-stt/app/api/openai_router.py

```
@@ -1,49 +1,45 @@
+router = APIRouter()
 
 @router.post(
     "/header-test",
     summary="Generate messages using OpenAI (streaming) - header test",
 )
-async def get_generated_messages_streaming(
+async def get_generated_messages_with_header(
     data: MessageRequest,
-    x_user_id: Union[str, None] = Header(default=None),
+    x_script_id: Union[str, None] = Header(default=None),
+    db: AsyncSession = Depends(get_db)
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)